upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -134,6 +134,12 @@ spec:
     metadata:
       labels:
         app: cluster-autoscaler
+      {{ if .Metrics }}
```

Comment:
These annotations are not much in use anymore and we tend not to set them. Instead we expect users to create pod/service monotors themselves.

Reply:
There are cases like [coreDNS](https://github.com/kubernetes/kops/blob/master/upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template#L231-L233) where these annotations seem useful. Do you suggest completely drop from here or migrating to a ServiceMonitor? I am cool with both options ^^

Reply:
Note that since Metrics is a pointer you need to use `WithDefaultBool`.